src/common/pico_time/time.c

```diff
@@ -13,9 +13,6 @@
 #include "hardware/sync.h"
 #include "hardware/gpio.h"
 
-CU_REGISTER_DEBUG_PINS(core)
-CU_SELECT_DEBUG_PINS(core)
-
 const absolute_time_t ABSOLUTE_TIME_INITIALIZED_VAR(nil_time, 0);
 // use LONG_MAX not ULONG_MAX so we don't have sign overflow in time diffs
 const absolute_time_t ABSOLUTE_TIME_INITIALIZED_VAR(at_the_end_of_time, ULONG_MAX);
```

Comment:
Hmm, the comment seems opposite to what the line below it is doing? :eyes: 
(yes, I realise this has nothing to do with the issue being fixed in this PR)